Ruby-Project/app.rb

```diff
@@ -0,0 +1,113 @@
+require_relative 'student_class'
+require_relative 'person_class'
+require_relative 'teacher_class'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'classroom'
+class App
+  attr_accessor :classrooms, :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @classrooms = []
+    @rentals = []
+  end
+
+  def all_people
+    @people.each_with_index do |p, index|
+      puts "#{index})" "[#{p.position}]" + " ID: #{p.id}" + " Name: #{p.name}" + " Age: #{p.age}"
+    end
+  end
+
+  def all_books
+    @books.each_with_index do |b, index|
+      puts "#{index})" + "Title: #{b.title}" + " Author: #{b.author}"
+    end
+  end
+
+  def books_rented
+    all_people
+    puts 'Chose the person by id'
+    person_id = gets.chomp.to_i
+    @people.find { |p| p.id == person_id }
+    puts "Rentals for Person ID #{person_id}:"
+    @rentals.each do |rental|
+      puts "Date: #{rental.date}, Book Title: "
+    end
```

Comment:
- [ Required ] I think it would be better if the `list rentals` menu option in the App works correctly. Currently, your design varies slightly from the video provided for this milestone. Please see the list below to improve the UX of the application.
   
   - [ ] The application should print the rentals of a given user, currently the application prints all the rentals even those that belong to another user. Please see the screenshot below and rectify this.
   
   - [ ] The rental being printed should also print the `name` of the author and the `title` of the book just the way it is in the video. Currently, these are not being printed. 

## Your design
![Screenshot from 2023-07-01 21-36-16](https://github.com/OnlyUnknown/OOP-school-library/assets/16392046/e77f25ec-f7b0-474d-84ad-df4557028b8b)

## Project wireframe
![Screenshot from 2023-07-01 21-52-52](https://github.com/OnlyUnknown/OOP-school-library/assets/16392046/6c8217d1-5a9d-4a21-acd0-f58ad743d706)

Reply:
I joined a call with the student and helped him fix the issue.